### PR TITLE
Update of the documentation, to show how to get a decoded JWT Token info from the current request

### PR DESCRIPTION
**Pull request by futbolsalas15**
I don't know if _index.md_ is the right file to document this, so I'm sorry if it is the wrong place. However, I really appreciate if you can document in some place, because it is a recurrent topic that some people (like me) don't find easily in Internet.

Thanks a lot!

### CODE REVIEW

**Comment by chalasr** [comment]
Nice PR, I'll look if I can find a better place for it, but be sure it will be in.
Thanks a ton

File: Resources/doc/index.md [hunk 1]
```diff
@@ -127,6 +127,28 @@ By default only the authorization header mode is enabled : `Authorization: Beare
 
 See [configuration reference](1-configuration-reference.md) document to enable query string parameter mode or change the header value prefix.
 
+### 3. (Optional) Get the decoded JWT Token from a Service/Controller (via Dependency Injection)
```
Comment (Oliboy50) [inline Resources/doc/index.md:130]:
```suggestion
### 3. (Optional) Get the decoded JWT from a Service/Controller (via Dependency Injection)
```

(T is for Token)

**Comment by fd6130** [comment]
@chalasr  I think i can move this to my PR #920 and it will be a new page under `Further documentation`, so this PR can be close after that. How do you think?

**Comment by chalasr** [comment]
@fd6130 Sounds good to me!

**Comment by fd6130** [comment]
> @fd6130 Sounds good to me!

There is an issue regarding to this PR.  Already create a new issue for this usage.

**Comment by fd6130** [comment]
Forgot to mention that this PR works for Guard but not the new authentication system. Using this PR example in latest version of this bundle will cause errors, i will make a PR for the fix soon.

**Comment by chalasr** [comment]
Closing in favor of #604. Thanks for raising @futbolsalas15!

**Pull request closed without merging**